web/src/components/shared/Ads/ConfirmRateChangeModal.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC } from 'react';
+import { Money, Currency } from '@bitzlato/money-js';
+import { Modal, ModalHeader, ModalBody } from 'web/src/components/ui/Modal';
+import { Text } from 'web/src/components/ui/Text';
+import { Stack } from 'web/src/components/ui/Stack';
+import { Box } from 'web/src/components/ui/Box';
+import { Button } from 'web/src/components/ui/Button';
+import { P2PMoneyFormat } from 'web/src/components/money/P2PFiatMoney';
+import { useSharedT } from 'web/src/components/shared/Adapter';
+import { parseNumeric } from 'web/src/helpers/parseNumeric';
+import LongDownArrowIcon from 'web/src/assets/svg/LongDownArrowIcon.svg';
+import LongUpArrowIcon from 'web/src/assets/svg/LongUpArrowIcon.svg';
+import LongRightArrowIcon from 'web/src/assets/svg/LongRightArrowIcon.svg';
+
+interface DiffRateProps {
+  prevRate: Money;
+  nextRate: Money;
+  cryptoCurrency: Currency;
+  isBuy: boolean;
+}
+
+const DiffRate: FC<DiffRateProps> = ({ prevRate, nextRate, cryptoCurrency, isBuy }) => {
+  const diff = nextRate.subtract(prevRate);
+  const procent = parseNumeric(nextRate.amount.div(prevRate.amount).sub(1).times(100).toString(), {
+    maxFractionDigits: 2,
+    allowNegativeNumeric: true,
+  });
+  const color =
+    (isBuy && diff.isPositive()) || (!isBuy && diff.isNegative()) ? 'danger' : 'success';
+  const plus = diff.isPositive() && '+';
```

Comment:
```suggestion
  const sign = diff.isPositive() ? '+' : '';
```

Reply:
А какие причины для этого?

Может так?
```suggestion
  const maybePlus = diff.isPositive() && '+';
```

Reply:
Подумал может минус может появится

Reply:
Тут смысл в подстановки плюса для положительных, а в отрицательных минус есть всегда

Reply:
Тогда ок